### ISSUE TITLE
Require Ctrl to trigger page preview on hover

### ISSUE THREAD

**Issue by g-adel** [pjeby/quick-explorer#80]
Hello, I'm struggling a bit with the page preview feature of Quick-Explorer. It can be quite distracting when looking for a file, and with some files like pdfs, the preview is actually large enough that it covers the link to the pdf, making it inaccessible.

I see there's a setting for toggling on the requirement of holding Ctrl to trigger Page Preview. However, I'm finding that whether I'm holding the Ctrl button or not (with the setting on or off), the preview shows up anyhow. I tried this on a fresh vault and found the exact same behavior. Let me know if you'd like any further details. Thanks!

**Comment by pjeby** [comment]
The Tab key toggles auto-preview on and off, and it's turned on by default whenever Obsidian starts or the plugin loads.  Once you turn it off it will stay off until the next restart of Obsidian or the plugin.

**Comment by j-maffe** [comment]
Ah... thanks for that! Is there no way to make the default off? It's a bit weird that there's a redundancy with the setting and that it can't be turned off permanently. Also, it might be a good idea to add this tooltip to the settings page :D
Thanks again!

**Comment by pjeby** [comment]
No, there is not a way to change the default.  Also, I don't control the settings page -- that's created by Obsidian.

**Comment by pjeby** [comment]
The settings I was referring to was the "Page Preview" core plugin settings, which is how you set whether Ctrl is required for a popup to appear in various places.  If you turn off QE's auto-hover (using Tab), then whether Ctrl is required is determined by that setting.  By turning on the toggle labeled "Quick Explorer" in that settings page, then QE will require Ctrl to hover preview (as long as you've initially turned the function off with Tab).

What keys *are* available on your tablet?

**Comment by j-adel** [comment]
Thanks for the clarification. The problem I'm facing is that I can't turn off the function with Tab. I, myself, don't even need hover for QE. It gets particularly bad on a windows tablet (Surface Go) because the hover actually covers over the QW menu itself; making half of the menu inaccessible under the preview window. 
The touchscreen keyboard has a Shift, Ctrl, and Enter for modifiers. I suppose I can try to find another touchscreen keyboard app. It's just that turning off (or just inverting) this feature would be a lot easier for my workflow.

If the problem is unsolvable on your side then it's of course no problem, thanks again.

**Comment by pjeby** [comment]
Well, I could add another hotkey for it.  You also could make a patched version of the plugin.  I could even add a setting of some kind.  It's just not something that I've needed myself or taken any time to do at present.

**Comment by j-adel** [comment]
If you could inform me how I can patch the extension or provide a setting for it, I'd be very grateful :D

**Comment by pjeby** [comment]
If you're able to build the plugin yourself, you can change this line: https://github.com/pjeby/quick-explorer/blob/34d1c868b4c88672caecbfad060cd02da4f13649/src/FolderMenu.ts#L43 to say `false` instead of `true`.  Otherwise, you'd need to edit the local `main.js`, looking for code like this:

```
var x=!0,P=class extends F{constructor(e,t,o,n){super(e);this.parent=e;this.folder=t;this.selectedFile=o;this.crumb=n;this.parentFolder=this.parent instanceof P?this.parent.folder:null;this.fileCount=e=>e instanceof c.TFolder?e.children.map(...
```

And change the `!0` to `!1`.  This is a harder change to make because 1) you mustn't change anything else, and 2) the `x=` bit might be another letter -- you have to search for the first occurrence of `this.parentFolder` and look backwards to just before the `class` to find the `!0`.

A future version of the plugin will include an option to set the default, along with probably some other options.

**Comment by j-adel** [comment]
That worked! Thanks a bunch!!

**Comment by Ai-Jani** [comment]
I was searching for this information, thank you so much for this great plugin too!

**Comment by greetclammy** [comment]
Piling on to request a permanent auto-preview disable option 🙏 The current tab-press solution resets on restart, and modifying plugin code is bound to break when the plugin updates.

**Comment by pjeby** [comment]
Fixed in [0.2.20](https://github.com/pjeby/quick-explorer/releases/tag/0.2.20) - the setting saves automatically, and there's a visible indicator toggle so you can see if it's on or off and change it.  The state is per-vault, per device and does not sync to other devices.
